src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreCommand.cs

```diff
@@ -486,7 +486,7 @@ private async Task<bool> AreCentralVersionRequirementsSatisfiedAsync(RestoreRequ
             // Log a warning if there are more than one configured source and package source mapping is not enabled
             if (restoreRequest.Project.RestoreMetadata.Sources.Count > 1 && !restoreRequest.PackageSourceMapping.IsEnabled)
             {
-                await _logger.LogAsync(RestoreLogMessage.CreateWarning(NuGetLogCode.NU1507, string.Format(CultureInfo.CurrentCulture, Strings.Warning_CentralPackageVersions_MultipleSourcesWithoutPackageSourceMapping, restoreRequest.Project.RestoreMetadata.Sources.Count)));
+                await _logger.LogAsync(RestoreLogMessage.CreateWarning(NuGetLogCode.NU1507, string.Format(CultureInfo.CurrentCulture, Strings.Warning_CentralPackageVersions_MultipleSourcesWithoutPackageSourceMapping, restoreRequest.Project.RestoreMetadata.Sources.Count, string.Join(", ", restoreRequest.Project.RestoreMetadata.Sources.Select(i => i.Name)))));
```

Comment:
Does the url matter more or the name?

Reply:
I'm not sure, I originally was calling the `.ToString()` method on `PackageSource` but it returns the format:

https://github.com/NuGet/NuGet.Client/blob/336320cc799d5a4e36da439148523e5ab9e4ff1e/src/NuGet.Core/NuGet.Configuration/PackageSource/PackageSource.cs#L186-L189

Which looks funky when the name and source are the same value.  Can I update the `ToString()` method to something like this instead:

```c#
public override string ToString() 
{
    if (Name == Source)
    {
        return Source;
    }
    else
    {
        return Name + " [" + Source + "]";
    }
} 
```

Or would that be considered a breaking change?  I think its useful to have both the name and the source but not if they are the same value.  I guess I could just add that logic here instead.  Thoughts?

Reply:
No clue what it breaks. 

We probably use the sources in a bunch of other error messages, we should probably match that. https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu1102

It uses the name. No one has really complained, maybe best we do not overthink it :) 

I'll approve. If you decide to go a different direction, happy to approve that as well.